src/commands/infra/init.test.ts

```diff
@@ -0,0 +1,45 @@
+import child_process from "child_process";
+import {
+  disableVerboseLogging,
+  enableVerboseLogging,
+  logger
+} from "../../logger";
+import { validatePrereqs } from "./init";
```

Comment:
Instead of this duplication can ew do the following?
`import { validatePrereqs, validateAzure, validateEnvVariables } from "./init";`
I'm not sure if this a Typescript best practice or not

Reply:
Yup! This can be simplified.